ConnorAndersonLarson/PlanetParty
# Testing/single page

# PR PlanetParty
**Connor A-L, Alex T & Katie B**

### What does this do?


### Are there any known bugs?


### Summary of the changes
Refactor/add tests to reflect Single Page view updates


### What ticket(s) do the changes resolve?
close #66 
Please Link Issues to this PR as necessary.
#66 
### Notes for the Reviewer
All tests should be passing, system is checking for unique planet numbers.


### How to Test Changes?
switch to branch, run `npm start` and `npm run cypress` in separate tabs in terminal, go through all four testing suites to ensure all are passing.
 

### Next Iterations
